### DISCUSSION TITLE
Add support for gRPC

### DISCUSSION THREAD

**Discussion by bibendi** [nginx-proxy/nginx-proxy#2503] [Ideas]
https://nginx.org/en/docs/http/ngx_http_grpc_module.html

**Comment by buchdag** [comment]
Hi.

[It's missing from the docs but it's already there.](https://github.com/nginx-proxy/nginx-proxy/blob/main/nginx.tmpl#L279-L283)

**Comment by bibendi** [comment]
Hey! Thank you!
